lib/driver.js

```diff
@@ -314,6 +321,8 @@ class XCUITestDriver extends BaseDriver {
       await retryInterval(15, 1000, async () => {
         log.debug('Sending createSession command to WDA');
         try {
+          // This is needed to put the '/status' output into the cache
+          await this.getStatus();
```

Comment:
I think it's OK to have it here - it's more obvious. Also, the '/status' call itself takes several milliseconds - I've checked it locally